HelpSource/Classes/Server.schelp

```diff
@@ -623,6 +623,15 @@ an optional instance of link::Classes/Condition:: used for evaluating this.
 argument:: ... args
 one or more valid OSC messages.
 
+method:: addr
+Returns the ip and TCP/UDP port being used by the specified server
+code::
+~myLocalServer=Server.local;
+~myLocalServer.addr;
+~myInternalServer=Server.internal;
+~myInternalServer.addr;
```

Comment:
@tiagmoraismorgado Please do not add examples using internal server (cf [James comment](https://github.com/supercollider/supercollider/pull/2582#issuecomment-268934642)).